### PR TITLE
Update code owners and maintainers

### LINKED ISSUE #1: Define Codeowners and Maintainers for the new repository
**Problem description**

There are currently no Codeowners and Maintainers defined for this new repository.

**Expected action**

Collect the initial Codeowners and Maintainers within https://github.com/camaraproject/APIBacklog/issues/70 and create a PR to update /CODEOWNERS and /MAINTAINERS.md accordingly.

**Additional context**

### PR DESCRIPTION
**Pull request by benhepworth**
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:

Add code owners and maintainers

#### Which issue(s) this PR fixes:

Fixes #1 

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Review by RandyLevensalor — APPROVED** [review]
LGTM

**Review by hdamker — COMMENTED** [review]
@benhepworth thanks for the PR. I've updated the @camaraproject/quality-by-design_codeowners team (proposed CODEOWNERS file is valid with that) and @camaraproject/quality-by-design_maintainers.

Before we can merge the PR, @shafayetkhan need to accept the invitation to the CAMARA Organization, and subscribe to the sub project mailing list https://lists.camaraproject.org/g/sp-coi

**Comment by hdamker** [comment]
> Before we can merge the PR, @shafayetkhan need to accept the invitation to the CAMARA Organization, and subscribe to the sub project mailing list https://lists.camaraproject.org/g/sp-coi

Welcome @shafayetkhan ... so we are done here.

**Review by hdamker — APPROVED** [review]

**Pull request merged**
